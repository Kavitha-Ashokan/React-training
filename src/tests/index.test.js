import React from "react";
import renderer from "react-test-renderer";
import App from "../index";

it("should render and match the snapshot", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('it should render a button', () => {
  const tree = renderer
    .create(<App text="Submit" type="submit" state="default" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

