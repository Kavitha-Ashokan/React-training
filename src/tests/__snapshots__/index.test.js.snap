// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render a button 1`] = `
<index
  state="default"
  text="Submit"
  type="submit"
/>
`;

exports[`should render and match the snapshot 1`] = `<index />`;
